tutorials/shaders/screen-reading_shaders.rst

```diff
@@ -102,56 +104,97 @@ With correct back-buffer copying, the two spheres blend correctly:
 
 .. warning:
 
-    Materials that use ``SCREEN_TEXTURE`` are considered transparent themselves and
-    will not appear in the resulting ``SCREEN_TEXTURE`` of other materials.
-    If you plan to instance a scene that uses a material with ``SCREEN_TEXTURE``,
+    In 3D materials that use ``hint_screen_texture`` are considered transparent themselves and
+    will not appear in the resulting screen_texture of other materials.
```

Comment:
```suggestion
    will not appear in the resulting screen texture of other materials.
```
Or with code markup if referring to the manually defined uniform, but that may be confusing?